source/directives/form.ts

```diff
@@ -0,0 +1,51 @@
+/// <reference path="../../node_modules/forms-js/dist/forms-js.d.ts" />
+
+module adaptor.directives {
+
+  export function FormDirective($compile, builder) {
+    var formsjsForm:formsjs.Form;
+
+    return {
+      restrict: 'AE',
+
+      scope: {
+        view: '=',
+        validation: '=',
+        model: '='
+      },
+
+      controller: function() {
+        formsjsForm = new formsjs.Form();
+
+        this.form = formsjsForm;
+      },
+
+      link: function($scope:any, $element:ng.IAugmentedJQuery, $attributes:ng.IAttributes) {
+        $scope.$watch('model', function(model:any) {
+          formsjsForm.formData = model;
+        });
+
+        $scope.$watch('validation', function(validation:formsjs.ValidationSchema) {
```

Comment:
That sounds like a reasonable solution for validation. The view schema might be a different beast though. But personally I don't like doing any kind of deep watch, since it might hurt performance, at least not as default anyway.